### DISCUSSION TITLE
RocketFuel 1.4.0 Released

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#2329] [Show and tell]
Theme upgraded to Bootstrap 5.2
New and improved NavBar

![RocketFuel 1 4 0-NavBar](https://user-images.githubusercontent.com/9447612/182403291-36b5f79b-d3af-4f53-b8ff-e89b36b85f11.gif)

**Comment by larsontim12** [comment]
Hi Leigh,

I just thought I would let you know that studio-elf.net website is coming up as malware.

Thanks,
Tim

Malwarebytes blocked a compromised site
Your Malwarebytes blocked this website because it may be compromised.

We strongly recommend you do not continue.

**Reply by leigh-pointer** [reply]
@larsontim12 thanks for the update.  Not sure why the cert is just renewed the only other thing changed was the NameServer.  Had a quick google and found this https://forums.malwarebytes.com/topic/91713-how-do-i-unblock-a-site/ if it helps?

**Comment by avazon** [comment]
Are there any code samples or docs how to use Rocket Fuel Theme? It looks great but I don't understand where I can learn more about how to use the package.

**Comment by leigh-pointer** [comment]
Have a look at https://www.studio-elf.net/oqtane/rocketfuel
________________________________
